tests/amor/pipeline_test.py

```diff
@@ -69,13 +69,19 @@ def test_run_data_pipeline(amor_pipeline: sciline.Pipeline):
 @pytest.mark.filterwarnings("ignore:Invalid transformation, missing attribute")
 def test_run_full_pipeline(amor_pipeline: sciline.Pipeline):
     amor_pipeline[SampleRotation[SampleRun]] = sc.scalar(0.85, unit="deg")
+    # Make the Q range cover a larger interval than the experiment is sensitive to.
+    # This let's us test the non-covered regions are filtered from the ORSO data.
+    amor_pipeline[QBins] = sc.geomspace(
+        dim="Q", start=0.005, stop=0.15, num=391, unit="1/angstrom"
+    )
```

Comment:
Would be nicer to put this in a separate test, instead of having one "let us test every aspect" one?

Reply:
Yeah I agree. But that would require re-running the pipeline again, making the tests slower.
So for that reason I opted for making it part of the test for the ORSO output instead.

Reply:
Since we both agree, maybe we can live with the slower tests? Alternatively, write a unit test for just this function, without running the full pipeline?

Reply:
> Since we both agree, maybe we can live with the slower tests?

That's a weird way of putting it. I agreed about the aspect you brought up. But taking into account the drawback mentioned I opted for another solution.

If you prefer a different trade-off, for example having slower but more well separated tests, we can do that instead.